addons/ophthalmology/stringtable.xml

```diff
@@ -7,188 +7,215 @@
             <Italian>Le coperture per occhi sono utilizzate per il trattamento di lesioni gravi all'occhio</Italian>
             <German>Ein Augenschild wird zur Behandlung von schweren Augenverletzungen verwendet</German>
             <Japanese>眼球保護帯は重傷となった目の治療に使用します</Japanese>
+            <Czech>Kryty očí se používají k léčbě těžkých poranění oka </Czech>
         </Key>
         <Key ID="STR_KAT_Ophthalmology_EyeCover_Item">
             <English>Eye Shield</English>
             <French>Protection oculaire</French>
             <Italian>Protezione oculare</Italian>
             <German>Augenschild</German>
             <Japanese>眼球保護帯</Japanese>
+            <Czech>Ochrana očí</Czech>
         </Key>
         <Key ID="STR_KAT_Ophthalmology_EyeCover_Left">
             <English>Left EyeCover</English>
             <French>Œil gauche Couvert</French>
             <Italian>Copertura occhio sinistro</Italian>
             <German>Linkes Augenschild</German>
             <Japanese>左眼球保護帯</Japanese>
+            <Czech>Levý Oční Štít</Czech>
         </Key>
         <Key ID="STR_KAT_Ophthalmology_EyeCover_Right">
             <English>Right EyeCover</English>
             <French>Œil droit Couvert</French>
             <Italian>Copertura occhio destro</Italian>
             <German>Rechtes Augenschild</German>
             <Japanese>右眼球保護帯</Japanese>
+            <Czech>Pravý Oční štít</Czech>
         </Key>
         <Key ID="STR_KAT_Ophthalmology_blink_action">
             <English>Blinking</English>
             <French>Clignotement</French>
             <Italian>Sbattito delle palpebre</Italian>
             <German>Blinzeln</German>
             <Japanese>まばたき</Japanese>
+            <Czech>Mrkání</Czech>
         </Key>
         <Key ID="STR_KAT_Ophthalmology_cba_name">
             <English>KAT - ADV Medical: Ophthalmology</English>
             <French>KAT - ADV Medical : Ophtalmologie</French>
             <Italian>KAT - ADV Medica: Oftalmologia</Italian>
             <German>KAT - ADV Medical: Augenheilkunde</German>
             <Japanese>KAM - 眼科 (Ophthalmology)</Japanese>
+            <Czech>KAT - ADV Zdravotní modul: Oftalmologie</Czech>
         </Key>
         <Key ID="STR_KAT_Ophthalmology_eyeInjuryPresent">
             <English>Eye Injury</English>
             <French>Lésions oculaires</French>
             <Italian>Ferita all'occhio</Italian>
             <German>Augenverletzung</German>
             <Japanese>眼外傷</Japanese>
+            <Czech>Oční zranění</Czech>
```

Comment:
`Poranění oka` sounds better imo. I am missing the context, but it just sounds better.